### PR TITLE
NEO-2325 // update story to show fix

### PR DESCRIPTION
**Pull request by joe-s-avaya**
[Link to updated Table story](https://deploy-preview-494--neo-react-library-storybook.netlify.app/?path=/story/components-table--embedded-selects)

**Before tagging the team for a review, I have done the following:**

- [x] run `yarn all` locally: ensures that all tests pass, formatting is done, types pass, and builds pass
- [x] reviewed my code changes
- [x] updated the Deploy Preview link at the top of this PR (or remove it if not applicable)
- [x] updated the Figma referense link at the top of this PR (or remove it if not applicable)
- [x] added any important information to this PR (description, images, GIFs, ect.)
- [x] done an accessibility check on my work (tested with Chrome's `axe Dev Tools`, Mac's VoiceOver, etc.)
- [ ] ~tagged `@avaya-dux/dux-design` if any visual changes have occurred~
- [x] tagged `@avaya-dux/dux-devs`

The actual bug was fixed in a previous PR, and I didn't realize I had an issue with my story code. I am resolving the story code in this PR. 

Note that there is a "HACK" note with a NEO Jira Issue to work on resolving that edge case.

`@avaya-dux/dux-devs`: review code

## Summary by CodeRabbit

- **New Features**
	- Enhanced the `EmbeddedSelects` component for better action selection handling.
	- Improved accuracy in reflecting the current state with an updated action selection mechanism.

- **Bug Fixes**
	- Implemented a safeguard to prevent redundant action executions based on the last action taken.

- **Improvements**
	- Enhanced data management during reset actions for clearer UI representation of user interactions.

### CODE REVIEW

File: src/components/Table/Table.stories.tsx [hunk 1]
```diff
@@ -929,6 +929,9 @@ export const EmbeddedSelects = () => {
 		action: null | string | string[],
 		row: IData,
 	) => {
+		// HACK: (NEO-2351) actions are being called twice, this fixes that issue
+		if (action === row.lastAction) return;
```
Comment (yangAtSpoken) [inline src/components/Table/Table.stories.tsx:933]: Does it mean one can not take the same action in a row?

**Review by yangAtSpoken — COMMENTED** [review]
one comment

File: src/components/Table/Table.stories.tsx (hunk 1 above)
Comment (joe-s-avaya) [reply src/components/Table/Table.stories.tsx:933]: Correct. Which is required by the use-case given by Yash.

Comment (yangAtSpoken) [reply src/components/Table/Table.stories.tsx:933]: ok

**Review by yangAtSpoken — APPROVED** [review]

**Pull request merged**